index.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>CV - Pinchuk Vladislav</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
+       div {
+        max-width: 1024px;
+        text-align: center;
+       } 
+    </style>
+</head>
+<body>
+    <div>
+    <header>
+        <h1>Pinchuk Vladislav Alexandrovich</h1>
+    </header>
+    <hr>
+    <img src="photo.jpg" width="150" alt="Photo Pinchuk Vladislav">
+     
+    <section>
+        <h3>Event Specialists / Event Manager <br>
+        <h4>Work experience - 11 years 8 months</h4>
+        </h3>
+        <p>Male, 30, born May 24, 1990</p>
+        <p><a href="tel:+375 (44) 515 49 83">+375 (44) 515 49 83</a><br>
+           <a href="mailto:freebka@gmail.com">freebka@gmail.com</a></p>
+        <p>Residence: Minsk <br>Citizenship: Belarus</p>
+    </section>
+
+    <section>
+        <h3>State institution Minskconcert</h3>
+        <p>July 2020 - present 3 months</p>
+        <h4>Head of the household</h4>
+        <article>Availability control in departments, purchase and delivery control of inventory items: drinking water, stationery, electrical household goods, household chemicals, consumables. Purchase of sound, light and computer equipment for the organization.
+            Inventory, accounting and write-off of material assets. Purchase of furniture, household and office equipment. Comprehensive organization of new jobs. Provision of personnel with the necessary equipment to carry out current work.</article>
```

Comment:
You can use as many new lines as you need in one tag. As you see this part of code is hard to read because of line width